crates/subspace-farmer/src/plot.rs

```diff
@@ -398,7 +385,7 @@ impl IndexHashToOffsetDB {
         let max_distance = {
             let mut iter = inner.raw_iterator();
             iter.seek_to_last();
-            iter.key().map(|key| *<&Sha256Hash>::try_from(key).unwrap())
+            iter.key().map(PieceDistance::from_big_endian)
```

Comment:
Hm, are you sure this should be big-endian? The data structure is little-endian by default and that is what used for check whether it fits max plot size or not in `subspace-solving` and `sc-consensus-subspace`.

Reply:
Sorry, didn't pay attention to it here. From correctness side it doesn't matter, so I will change to little endian.

Reply:
Actually it does matter, as data gets sorted by rocksdb here. So it should be big endian 😅

Reply:
Hm, if we actually want this natural sorting to work, we do need big-endian I think, we just need to make sure conversion is the same everywhere.

Reply:
No, we store the distance in rocksd, so it would matter, as in big endian case seeking to last item will always return the maximum distance. In little endian case it will return some gibberish.